SPECS/nmap/nmap.signatures.json

```diff
@@ -1,5 +1,6 @@
 {
  "Signatures": {
-  "nmap-7.92.tar.bz2": "a5479f2f8a6b0b2516767d2f7189c386c1dc858d997167d7ec5cfc798c7571a1"
+  "nmap-7.93.tar.bz2": "55bcfe4793e25acc96ba4274d8c4228db550b8e8efd72004b38ec55a2dd16651",
+  "remove_openssl_macro.patch": "67a713b2597c1d44bf18d72f3165be54d7f840637d6a15e1c8f54121b27d1f29"
```

Comment:
I don't believe you need to include the patch file and signature here. Since it is in the spec directory, the .patch file will be picked up automatically at time of build. The signatures file is only for things that are pulled from the blob store afaik.